### DISCUSSION TITLE
Raspotify does not send audio on the hdmi output

### DISCUSSION THREAD

**Discussion by cacy82** [dtcooper/raspotify#487] [Q&A] [answered]
Hi,

I installed raspotify to use it on the magic mirror, if I connect the headphones to the 3.5mm jack it goes without problems, but if I move the audio on the raspbian to the hdmi output (I connected the raspberry to a monitor with built-in speakers), all the sounds I hear them on the monitor speakers, except for raspotify which remains on the headphones.
I state that I tried to access the / etc / raspotify file to see if there was anything to change in the settings, but I realized that this file was not created at the time of installation and therefore does not exist.

How can I solve the problem without having to buy two speakers to connect to the 3.5mm socket?

**Comment by JasonLG1979** [comment, marked as answer]
Please read the wiki before asking questions. There is a [Troubleshooting](https://github.com/dtcooper/raspotify/wiki/Troubleshooting) page which has a section to tell you how to configure the audio output.

**Reply by cacy82** [reply]
Oh Thanks! Problem resolved! That section had escaped me!
